docs/api.md

```diff
@@ -327,6 +336,14 @@ Arbitrary combinations of individual platforms are also supported via a comma-de
 The non-`all` values correspond to the platform names used by [Electron releases]. This value
 is not restricted to the official set if [`download.mirror`](#download) is set.
 
+##### `prebuiltAsar`
+
+*String*: (default: `undefined`)
+
+A path to a pre-built asar file. 
+
+**NOTE:** This option circumvents many options such as `asar`, `afterCopy`, `afterPrune`, `ignore`, `prune`, or any option related to the copy processes. These options are for generating an app directory/asar with this tool. However, if a prebuilt asar file is provided, these options are not needed.
```

Comment:
I'd get rid of `, or any option related to the copy processes`. Also, it's a little too verbose in general, but I need some time to think of a better way to word it.